hack/run-e2e-kind.sh

```diff
@@ -30,17 +30,13 @@ export CLUSTER_CONTEXT="--name ${CLUSTER_NAME}"
 
 export KIND_OPT=${KIND_OPT:="--config ${VK_ROOT}/hack/e2e-kind-config.yaml"}
 
-function get-k8s-server-version {
-    echo $(kubectl version --short=true | grep Server | sed "s/.*: v//" | tr "." " ")
```

Comment:
How adout just remove `--short`?

Reply:
we can not get major and minor number if just remove --short：
we get client version along with server version  by newer kubectl.  So we should get major and minor of server version from these information by shell or commands.